packages/replay-next/components/context-menu/ContextMenu.tsx

```diff
@@ -1,10 +1,13 @@
-import { MouseEvent, ReactNode, useRef } from "react";
+import { MouseEvent, ReactNode, useLayoutEffect, useRef, useState } from "react";
 import { createPortal } from "react-dom";
 
 import useModalDismissSignal from "replay-next/src/hooks/useModalDismissSignal";
 
 import styles from "./ContextMenu.module.css";
 
+// TODO Should we be using clientX/clientY instead?
+// I guess it doesn't matter since Replay doesn't scroll.
```

Comment:
I think this is out of scope of this PR but I wanted to leave it as a thought for later.